platform/logging/LogLevel.ts

```diff
@@ -0,0 +1,55 @@
+import { assertNever } from '../lib/utils';
+
+export type LogLevelId =
+  | 'all'
+  | 'fatal'
+  | 'error'
+  | 'warn'
+  | 'info'
+  | 'debug'
+  | 'trace'
+  | 'off';
+
+export class LogLevel {
+  static Off = new LogLevel('off', 1);
+  static Fatal = new LogLevel('fatal', 2, 'red');
```

Comment:
I'm planning to get rid of `color` from the `LogLevel` since it seems to be only related to `ConsoleAppender`.